papra/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: papra
+name: Papra
+tagline: The minimalistic document archiving platform
+category: files
+version: "25.10.0"
+description: >-
+  Papra is a minimalistic document management and archiving platform. It is designed to be simple to use and accessible to everyone. 
+  
+  
+  Papra is a platform for long-term document storage and management, like a digital archive for your documents.
+
+
+  Forget about that receipt of that gift you bought for your friend last year, or that warranty for your new phone. With Papra, you can easily store, forget, and retrieve your documents whenever you need them.
+developer: Papra
+website: https://github.com/papra-hq
```

If needed, their's a dedicated website

```suggestion
website: https://papra.app
```